generate_cloudbuild.py

```diff
@@ -121,15 +111,29 @@ def MakeGcrTags(label_without_tag,
                         gcrio_suffix=gcr_suffix,
                         maybe_hypen=maybe_hypen,
                         label=label_with_tag))
+        if arch:
+            t.append(
+                '\'{gcrprefix}/{gcrio_project}/{gcrio_suffix}/{rebrand_name}:$TAG_NAME{maybe_hypen}{label}-$_DATE-{arch}\''
+                .format(gcrprefix=gcr_prefix,
+                        gcrio_project=GCRIO_PROJECT,
+                        gcrio_suffix=gcr_suffix,
+                        rebrand_name=REBRAND_NAME,
+                        maybe_hypen=maybe_hypen,
+                        label=label_without_tag,
+                        arch=arch))
     return t
 
 # Make all the tags and save them
 tags={}
 multi_arch_tags={}
+multi_arch_arm_tags={}
+multi_arch_amd_tags={}
```

Comment:
Defined twice

Reply:
Actually one is for arm images and the other is for amd.

Reply:
Ah I see. Ack.